### ISSUE TITLE
Feereport wrong day_fee_sum and week_fee_sum

### ISSUE THREAD

**Issue by sendbitcoin** [lightningnetwork/lnd#4639]
### Background
I ran lncli feereport and got his result:

![feereport1](https://user-images.githubusercontent.com/60074674/93930017-fe904200-fd1c-11ea-9439-30bca2320c35.png)

a couple hours and 2 routing events later i ran it again and got this result:

![feereport2](https://user-images.githubusercontent.com/60074674/93930173-3d25fc80-fd1d-11ea-9e14-9ace83b1869d.png)

how can the month fee have gone up but the daily and weekly fee sum have stayed the same?

shouldnt all three have increase by 2 instead of just the month?

**Comment by guggero** [comment]
Are you running on Windows?

**Comment by sendbitcoin** [comment]
@guggero No this is LND 0.11.0-beta running inside btcpay server docker

**Comment by guggero** [comment]
Okay, then the specific bug I was thinking of isn't relevant. Not sure what's going on here then.

**Comment by sendbitcoin** [comment]
@guggero ok I figured it out... I actually considered this yesterday but thought it's way too unlikely:

The rolling total of both the daily and the weekly fee report just happened to perfectly line up so that 2 sats came in one end while 2 sats fell out the other end for both in those couple hours.

I loaded all the routing events with time and amount into excel and double checked everything.
Everything adds up perfectly and it's really the case of the rolling balance being perfetly lined up so it looks like nothing changed when it's actually showing 2 "different" sats. 

Closing the issue as there is no bug.
